pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -552,8 +553,8 @@ def newpkgcb(pkgId, name, arch):
 
         # TODO: handle parsing errors/warnings, warningcb callback can be used below
         cr.xml_parse_primary(primary_xml_path, pkgcb=pkgcb, do_files=False)
-        cr.xml_parse_filelists(filelists_xml_path, newpkgcb=newpkgcb)
-        cr.xml_parse_other(other_xml_path, newpkgcb=newpkgcb)
+        # cr.xml_parse_filelists(filelists_xml_path, newpkgcb=newpkgcb)
```

Comment:
If we're going to leave these in here as comments, we need to have a comment RE *why* they're still here, and preferably a related-issue describing the Thing that will happen that will let us un-comment them.